src/GitHub.jl

```diff
@@ -4,19 +4,19 @@ module GitHub
 
 using Compat
 using Compat.Dates
-
 if VERSION >= v"0.7.0-DEV.2338"
     using Base64
 end
 
+const ?{T} = Union{T, Nothing}
```

Comment:
I don't think this is allowed on 0.7. IIRC Carlo Baldassi submitted a PR that removes the parsing of `?` as an identifier.

Reply:
If you want a shorthand for `Union{T, Nothing}`, I recommend something like `Maybe{T}`.

Reply:
Yeah, I'll change this.